### DISCUSSION TITLE
Can I create a network with just a phone and a sensor device?

### DISCUSSION THREAD

**Discussion by phoebebright** [openthread/openthread#7808] [Q&A] [answered]
I'm wondering if I can create a thread network using just a custom made sensor device (end device) and a mobile phone (router + border router) that can upload data via the phone's wifi?  I'm wanting to build a device that acts as a stationary end device in the home but can be taken out and about with the phone and sensor data will continue to be uploaded to a cloud seamlessly.

**Comment by jwhui** [comment, marked as answer]
Thread is currently defined to only communicate via IEEE 802.15.4 radios. or the sensor/end device to communicate directly to the mobile phone, the mobile phone would also need to support IEEE 802.15.4. If the mobile phone supports IEEE 802.15.4, then it is possible for the mobile phone to serve as a Thread border router.

**Reply by phoebebright** [reply]
Thanks, that very helpful.  Sounds like I am going in the wrong direction trying to keep to Thread in a mobile environment and will think about be able to switch to Bluetooth when Thread is not available.
